projects/packages/jwt/composer.json

```diff
@@ -0,0 +1,55 @@
+{
+	"name": "automattic/jetpack-jwt",
+	"description": "A Simple JWT package",
```

Comment:
I'd recommend adding more details here a bit, as well as in the readme, so in the future new and existing contributors, and AI tools, can find that package whenever we need something like this.